.github/workflows/grahpql_docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Update GraphQL Docs
+
+on:
+  schedule:
+    - cron: "51 */1 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: build docs
+        env:
+          KONTIST_CLIENT_ID: ${{secrets.DOCS_CLIENT_ID}}
+          KONTIST_CLIENT_SECRET: ${{secrets.DOCS_CLIENT_SECRET}}
+          KONTIST_USERNAME: ${{secrets.DOCS_USER}}
+          KONTIST_PASSWORD: ${{secrets.DOCS_PASSWORD}}
+          KONTIST_CLIENT_SCOPES: accounts,users,transactions,transfers,subscriptions,statements
+        run: |
+          ./update_docs.sh $(npx kontist token)
```

Comment:
should it be `./update_docs.sh "Bearer ${npm kontist token}"` 🤔 or maybe we should update the shell script to append the "Bearer " string directly 🤔

Reply:
Good point, will update `update_docs.sh`